.gitignore

```diff
@@ -45,3 +45,5 @@ projects/angular-resizable-element/package.json
 # System Files
 .DS_Store
 Thumbs.db
+package-lock.json
+.gitignore
```

Comment:
nit: can you remove the `.gitignore` from this file as we don't want it to ignore itself 😄